package.json

```diff
@@ -74,18 +74,15 @@
     "lint": "npm run lint:js && npm run lint:solidity",
     "lint:js": "eslint --max-warnings 0 .",
     "lint:solidity": "slippy 'contracts/**/*.sol'",
-    "dev": "nodemon --ext js,sol -x hardhat test 'test/unit/**/*.js' 'test/integration/**/*.js' 'test/layout/*.js' --show-stack-traces",
-    "dev-integration": "nodemon --ext js,sol -x hardhat test 'test/integration/**/*.js' --show-stack-traces",
-    "dev-unit": "nodemon --ext js,sol -x hardhat test 'test/unit/**/*.js' --show-stack-traces",
-    "dev-ui": "hardhat run scripts/local-deploy.js",
+    "dev": "nodemon --ext js,sol -x hardhat test test/unit test/integration test/layout --show-stack-traces",
+    "dev-integration": "nodemon --ext js,sol -x hardhat test test/integration --show-stack-traces",
+    "dev-unit": "nodemon --ext js,sol -x hardhat test test/unit --show-stack-traces",
     "coverage": "NODE_OPTIONS='--max-old-space-size=8192' hardhat coverage --testfiles 'test/{unit,integration}/**/*.js'",
-    "test": "hardhat test 'test/unit/**/*.js' 'test/integration/**/*.js' 'test/layout/*.js'",
-    "test-integration": "hardhat test 'test/integration/**/*.js'",
-    "test-unit": "hardhat test 'test/unit/**/*.js'",
-    "test-layout": "hardhat test 'test/layout/*.js'",
+    "test": "hardhat test test/unit test/integration test/layout",
```

Comment:
`test/layout` dir does not exist maybe we should drop it for now so the test script passes in the CI